tests/projects/PlasoFailPath1/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's make a test_data directory, and exclude it from codefactor/pylint and put these there.